core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -145,6 +145,16 @@ private[spark] object UIUtils extends Logging {
     }
   }
 
+  def formatBytes(bytes: Long): String = {
```

Comment:
FYI, there is `Utils.bytesToString` already, but it is using decimal units. In this case it makes sense to format in binary units, as you are trying to match what is shown in the UI for the search. You should probably add a comment above to say it's purpose. How about changing the name to `formatBytesBinary`?

Reply:
Makes sense, have updated the code. Thank you.